tests/test_views.py

```diff
@@ -116,5 +122,49 @@ def test_response_404_when_none_specified(self):
             content = json.loads(response.content.decode('utf-8'))
             self.assertCountEqual({'message': 'No checks found', 'error': 404}, content)
 
+    @override_settings(WATCHMAN_LOGIN=False)
+    def test_login_not_required(self):
+        # Reload settings - and all dependent modules - from scratch
+        reload(sys.modules['watchman.settings'])
+        reload(sys.modules['watchman.decorators'])
+        reload(sys.modules['watchman.views'])
+
+        request = RequestFactory().get('/')
+
+        response = views.status(request)
+
+        self.assertEqual(response.status_code, 200)
+
+    @override_settings(WATCHMAN_LOGIN=True)
+    def test_response_when_login_required_is_redirect(self):
+        # Reload settings - and all dependent modules - from scratch
+        reload(sys.modules['watchman.settings'])
+        reload(sys.modules['watchman.decorators'])
+        reload(sys.modules['watchman.views'])
+
+        request = RequestFactory().get('/')
+        request.user = AnonymousUser()
+        request.user.is_authenticated = lambda: False
```

Comment:
According to [the `AnonymousUser` docs](https://docs.djangoproject.com/en/1.8/ref/contrib/auth/#anonymous-users), `is_authenticated` should already be `False`, so is there a reason we need to do this as well?